compat/src/index.js

```diff
@@ -174,6 +174,13 @@ function createElement(...args) {
 
 	let type = vnode.type, props = vnode.props;
 	if (typeof type!='function') {
+		Object.keys(props).forEach(key => {
+			if (/^on(Ani|Tra)/.test(key)) {
```

Comment:
Quick question: Is this true for all `onAnimation*` and all `onTransition*` events?

Reply:
Hmm only for start and end I presume we could be taking capture here too

Reply:
Hey @JoviDeCroock I was looking over our `compat` code and there's a `applyEventNormalization` function. Should this be part of that function? I'm thinking it'll be easier to tackle updates on that code in the future since all the compat event handlers would be in one place, maybe?